### PR TITLE
Searching with gem ransack part 1

### PR DESCRIPTION
**Pull request by ghost**
image

### CODE REVIEW

**Comment by ghost** [comment]
redmine ticket https://edu-redmine.sun-asterisk.vn/issues/11516

**Comment by ghost** [comment]
ready

File: app/views/trainer/courses/_search_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      <%= f.text_field :name_cont, class: "form-control", placeholder:
+      t(".name_search") %>
+    </div>
+    <div class="form-group form-component">
+      <%= f.select :status_eq, Course.statuses, {prompt: t(".status_search")},
+      class: "form-control" %>
+    </div>
+    <%= f.submit t(".search_button"), class: "btn btn-default form-component" %>
+    <%= link_to t(".reset_link"), request.path, class:"form-component" %>
+  <% end%>
```
Comment (VuThiTranVan) [inline app/views/trainer/courses/_search_form.html.erb]: ` <% end %>`

File: app/views/trainer/trainees/_search_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="trainee-search-form">
+  <%= search_form_for [:trainer ,@search], url: trainer_trainees_path, class: "form-inline" do |f| %>
+    <div class="form-group ">
+      <%= link_to t(".reset_link"), request.path, class: "cancel-search" %>
+      <%= f.text_field :name_or_email_cont_any, class: "form-control", placeholder:
+      t(".name_search") %>
+    </div>
+    <%= f.submit t(".search_button"), class: "btn btn-default" %>
+  <% end%>
```
Comment (VuThiTranVan) [inline app/views/trainer/trainees/_search_form.html.erb]: `<% end %>` thiếu space sau end đấy

File: app/views/trainer/trainees/_search_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="trainee-search-form">
+  <%= search_form_for [:trainer ,@search], url: trainer_trainees_path, class: "form-inline" do |f| %>
+    <div class="form-group ">
+      <%= link_to t(".reset_link"), request.path, class: "cancel-search" %>
+      <%= f.text_field :name_or_email_cont_any, class: "form-control", placeholder:
+      t(".name_search") %>
```
Comment (VuThiTranVan) [inline app/views/trainer/trainees/_search_form.html.erb]: indent

File: app/views/trainer/courses/_search_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<div class="course-search-form">
+  <%= search_form_for [:trainer ,@search], class: "form-inline" do |f| %>
+    <div class="form-group ">
+      <%= f.text_field :name_cont, class: "form-control", placeholder:
+      t(".name_search") %>
+    </div>
+    <div class="form-group form-component">
+      <%= f.select :status_eq, Course.statuses, {prompt: t(".status_search")},
+      class: "form-control" %>
```
Comment (VuThiTranVan) [inline app/views/trainer/courses/_search_form.html.erb]: indent

**Comment by ghost** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Pull request merged**